code/obj/item/uplinks.dm

```diff
@@ -432,6 +432,13 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 					if (istype(antagonist_role) && !istype(I, /datum/syndicate_buylist/generic/telecrystal))
 						antagonist_role.purchased_items.Add(I)
 
+				if (src.purchase_flags & UPLINK_NUKE_OP)
+					var/datum/antagonist/nuclear_operative/antagonist_role = usr.mind?.get_antagonist(ROLE_NUKEOP)
+					if (antagonist_role == null )
+						antagonist_role = usr.mind?.get_antagonist(ROLE_NUKEOP_COMMANDER)
```

Comment:
```suggestion
					var/datum/antagonist/nuclear_operative/antagonist_role = usr.mind?.get_antagonist(ROLE_NUKEOP) || usr.mind?.get_antagonist(ROLE_NUKEOP_COMMANDER)
```
You can do this due to boolean shortcutting, a little bit neater (also applies to other instances of this)